endpoints/npm.ts

```diff
@@ -159,3 +164,30 @@ const parseDependents = html => {
   if (depLink.length !== 1) return -1
   return depLink.text().replace(/[^0-9]/g, '')
 }
+
+async function typesDefinition(pkg: string, tag: string = 'latest') {
+    const endpoint = `https://cdn.jsdelivr.net/npm/${ pkg }@${ tag }/package.json`
+    let meta = await got(endpoint).then(res => res.body)
+
+    if (typeof meta.types === 'string' || typeof meta.typings === "string") {
+        return {
+            subject: 'types',
+            status: 'included',
+            color: '0074c1'
+        }
+    }
+
+    const typesPkg = '@types/' + (pkg.charAt(0) === "@" ? pkg.slice(1).replace('/', '__') : pkg)
+    const typesEndpoint = `https://cdn.jsdelivr.net/npm/${ typesPkg }@latest/package.json`
```

Comment:
I think you should pass `tag` here too. We want the `@types` package for the given tag. It may be a rare case, but a possibility.